app/assets/javascripts/discourse/app/widgets/search-menu.js

```diff
@@ -22,10 +26,8 @@ export function initSearchData() {
   searchData.results = {};
   searchData.noResults = false;
   searchData.term = undefined;
-  searchData.typeFilter = null;
+  searchData.typeFilter = "exclude_topics";
```

Comment:
I'm wondering if we should move this to a constant like `DEFAULT_TYPE_FILTER` so that we don't duplicate this string across multiple places making it hard to change in the future.